black/Dockerfile

```diff
@@ -1,11 +1,13 @@
 FROM python:3.7-alpine
 
-RUN addgroup -S black && adduser -D -S black black && \
-    pip install black==19.3b0 && \
+RUN apk add --no-cache --virtual=.build-deps build-base && \
+    addgroup -S black && adduser -D -S black black && \
+    pip install --no-cache-dir black==19.10b0 && \
```

Comment:
Just a consideration: what about moving black version to the top:

```dockerfile
ENV BLACK_VERSION=19.10b0
```

And that using the env variable here and in `LABEL` command?
So that people don't forget to change it in both places in the future...

Reply:
Agree